update_sources.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python2.7
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+import argparse
+import shutil
+import subprocess
+
+'''
+This fork needs to track updates in the real MobileCoin SDK. 
+This script can be used to copy over the files from the real SDK. 
+It does not update the podspec as well; that must be done manually.
+
+To update this fork to reflect a new version of the MobileCoin SDK:
+
+* Ensure `MobileCoin-Swift` repo is cloned in sibling directory and is at the correct commit.
+* Ensure `libmobilecoin-ios-artifacts` repo is cloned in sibling directory and is at the correct commit.
+* Run this script.
+* Edit the MobileCoinMinimal.podspec.
+  * You probably only need to update the version number.
+* Commit and push these changes in a branch.
+* Update Signal-iOS to use the new branch.
+  * Edit Podfile to reflect new branch.
+  * bundle install;bundle exec pod update MobileCoinMinimal
+* Make sure MobileCoinHelperTests is passing.
+'''
+
+git_repo_path = os.path.abspath(subprocess.check_output(['git', 'rev-parse', '--show-toplevel']).strip())
+
+def copy_path(path, src_repo_name, dst_prefix = None):
+
+    print 'Copying:', path
+
+    src_repo_path = os.path.abspath(os.path.join(git_repo_path, "..", src_repo_name))
+    if not os.path.exists(src_repo_path):
+        print("Source repo not found:", src_repo_path)
+
+    src_file_path = os.path.join(src_repo_path, path)
+    if not os.path.exists(src_file_path):
+        print("Source file not found:", src_file_path)
+
+    dst_file_path = git_repo_path
+    if dst_prefix is not None:
+        dst_file_path = os.path.join(dst_file_path, dst_prefix)
+    dst_file_path = os.path.join(dst_file_path, path)
+    if os.path.exists(dst_file_path):
+        print("Destination file already exists:", dst_file_path)
+
+    dst_parent_path = os.path.dirname(dst_file_path)
+    if not os.path.exists(dst_parent_path):
+        os.makedirs(dst_parent_path)
+    
+    # shutil.copyfile(src_file_path, dst_file_path)
+    
+    lines_to_remove = set([
+        "import LibMobileCoin",
+        ])
+    substrings_to_replace = {
+        # 'MobileCoin.Scheme': 'Scheme',
+        'MobileCoin.Scheme': 'MobileCoinProtos.Scheme',
+        # '@testable import MobileCoin': '@testable import MobileCoinProtos',
+        'import MobileCoin': 'import MobileCoinProtos',
+        ': XCTestCase {': ': MobileCoinProtosSwiftTest {',
```

Comment:
We update the source files as they are copied, replacing certain substrings.  Most of this didn't end up being necessary in the final fork.